app/commands.py

```diff
@@ -504,14 +504,21 @@ def populate_redis_template_usage(service_id, day):
     Recalculate and replace the stats in redis for a day.
     To be used if redis data is lost for some reason.
     """
+    # the day variable is set to midnight of that day
     assert current_app.config['REDIS_ENABLED']
```

Comment:
Do we actually want to assert here in app/commands?

Reply:
changed to if statement and `sys.exit(1)`